### PR TITLE
[WIP]Create run_docker_local_example.sh

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: ppml/build_docker_image.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export HTTP_PROXY_HOST=$1 #your_http_proxy_host
```
Comment (glorysdj) [inline ppml/build_docker_image.sh:1]: we may define a build-environments.sh, and source it in build-docker-image.sh
this should be named as build-docker-image.sh
and put in under the /ppml/trusted-cluster-serving/scala/docker-graphene/

File: ppml/run_docker_create_keys.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+mkdir keys && cd keys
```
Comment (glorysdj) [inline ppml/run_docker_create_keys.sh:1]: named as generate-keys.sh?

File: ppml/run_docker_store_pwd.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+mkdir password && cd password
```
Comment (glorysdj) [inline ppml/run_docker_store_pwd.sh:1]: named as generate-password.sh?

Comment (glorysdj) [reply ppml/run_docker_store_pwd.sh:1]: and also pass the "YOUR_PASSWORD" as input argument

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md [hunk 4]
```diff
@@ -3,72 +3,30 @@ Please mind the ip and file path settings, they should be changed to the ip/path
...
-    --build-arg https_proxy=http://$HTTPS_PROXY_HOST:$HTTPS_PROXY_PORT \
-    --build-arg HTTP_PROXY_HOST=$HTTP_PROXY_HOST \
-    --build-arg HTTP_PROXY_PORT=$HTTP_PROXY_PORT \
-    --build-arg HTTPS_PROXY_HOST=$HTTPS_PROXY_HOST \
-    --build-arg HTTPS_PROXY_PORT=$HTTPS_PROXY_PORT \
-    --build-arg JDK_VERSION=8u192 \
-    --build-arg JDK_URL=$JDK_URL \
-    --build-arg no_proxy=x.x.x.x \
-    -t intelanalytics/analytics-zoo-ppml-trusted-cluster-serving-scala-graphene:0.10-SNAPSHOT -f ./Dockerfile .
+./build_docker_image.sh your_http_proxy_host your_http_proxy_port your_https_proxy_host your_https_proxy_port http://your-http-url-to-download-jdk
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/README.md:6]: simply tell user to modify build-environments.sh
and run  build-docker-image.sh

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md [hunk 5]
```diff
@@ -3,72 +3,30 @@ Please mind the ip and file path settings, they should be changed to the ip/path
...
-    openssl req -new -key server.key -out server.csr
-    openssl x509 -req -days 9999 -in server.csr -signkey server.key -out server.crt
-    cat server.key > server.pem
-    cat server.crt >> server.pem
-    openssl pkcs12 -export -in server.pem -out keystore.pkcs12
-    keytool -importkeystore -srckeystore keystore.pkcs12 -destkeystore keystore.jks -srcstoretype PKCS12 -deststoretype JKS
-    openssl pkcs12 -in keystore.pkcs12 -nodes -out server.pem
-    openssl rsa -in server.pem -out server.key
-    openssl x509 -in server.pem -out server.crt
+./run_docker_create_keys.sh
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/README.md:15]: change sh name

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md [hunk 6]
```diff
@@ -3,72 +3,30 @@ Please mind the ip and file path settings, they should be changed to the ip/path
...
 ```
-You also need to store the password you used in previous step in a secured file:
-```
-    mkdir password && cd password
-    openssl genrsa -out key.txt 2048
-    echo "YOUR_PASSWORD" | openssl rsautl -inkey key.txt -encrypt >output.bin
+You also need to store the password you used in previous step in a secured file: <br>
+```bash
+./run_docker_store_pwd.sh
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/README.md:19]: change sh name
and tell user to pass the password used in previous step to generate-password.sh

File: ppml/trusted-cluster-serving/scala/docker-graphene/build_docker_image.sh [hunk 7]
```diff
@@ -0,0 +1,16 @@
+export HTTP_PROXY_HOST=$1 #your_http_proxy_host
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/build_docker_image.sh:1]: why there are two build docker image sh?

File: ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_create_keys.sh [hunk 8]
```diff
@@ -0,0 +1,11 @@
+mkdir keys && cd keys
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_create_keys.sh:1]: and one public generate-keys.sh, and put in /ppml

File: ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_local_example.sh [hunk 9]
```diff
@@ -0,0 +1,21 @@
+export KEYS_PATH=$1 #the_dir_path_of_your_prepared_keys
+export SECURE_PASSWORD_PATH=$2 #the_dir_path_of_your_prepared_password
+export LOCAL_IP=$3 #your_local_ip_of_the_sgx_server
+sudo docker run -itd \
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_local_example.sh:4]: We may just put KEYS_PATH=the_dir_path_of_your_prepared_keys,... etc here
and ask user to modify these before run the script
named as run-docker-local-example.sh?

File: ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_store_pwd.sh [hunk 10]
```diff
@@ -0,0 +1,3 @@
+mkdir password && cd password
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_store_pwd.sh:1]: only need one generate-password.sh and put it in /ppml

File: ppml/build_docker_image.sh (hunk 1 above)
Comment (ManfeiBai) [reply ppml/build_docker_image.sh]: > this should be named as build-docker-image.sh
> and put in under the /ppml/trusted-cluster-serving/scala/docker-graphene/
> just export HTTP_PROXY_HOST=your_http_proxy_host, ... etc.
> we can ask user to modify these in the README

Thanks, done~

File: ppml/run_docker_create_keys.sh (hunk 2 above)
Comment (ManfeiBai) [reply ppml/run_docker_create_keys.sh]: Done~

File: ppml/run_docker_store_pwd.sh (hunk 3 above)
Comment (ManfeiBai) [reply ppml/run_docker_store_pwd.sh]: Tks, done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md (hunk 5 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/README.md]: done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md (hunk 6 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/README.md]: done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/build_docker_image.sh (hunk 7 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/build_docker_image.sh]: no build-docker-image.sh

Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/build_docker_image.sh]: no build-docker-image.sh under ppml dir

File: ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_create_keys.sh (hunk 8 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_create_keys.sh]: done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_local_example.sh (hunk 9 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_local_example.sh]: tks, done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_store_pwd.sh (hunk 10 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/run_docker_store_pwd.sh]: tks, done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md [hunk 11]
```diff
@@ -2,72 +2,32 @@
...
 ```
-You also need to store the password you used in previous step in a secured file:
-```
-    mkdir password && cd password
-    openssl genrsa -out key.txt 2048
-    echo "YOUR_PASSWORD" | openssl rsautl -inkey key.txt -encrypt >output.bin
+You also need to store the password you used in previous step in a secured file: <br>
+This script in also under /ppml: <br>
+```bash
+./generate-password.sh
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/README.md:21]: should pass your_password?

File: ppml/trusted-cluster-serving/scala/docker-graphene/build-docker-image.sh [hunk 12]
```diff
@@ -0,0 +1,17 @@
+export HTTP_PROXY_HOST=your_http_proxy_host
+export HTTP_PROXY_PORT=your_http_proxy_port
+export HTTPS_PROXY_HOST=your_https_proxy_host
+export HTTPS_PROXY_PORT=your_https_proxy_port
+export JDK_URL=http://your-http-url-to-download-jdk
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-graphene/build-docker-image.sh:5]: your_http_url_to_download_jdk?

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md (hunk 4 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/README.md]: tks, done~

File: ppml/trusted-cluster-serving/scala/docker-graphene/README.md (hunk 11 above)
Comment (ManfeiBai) [reply ppml/trusted-cluster-serving/scala/docker-graphene/README.md]: tks,done~

**Pull request merged**